### ISSUE TITLE
Add inner and outer mirroring of objects

### ISSUE THREAD

**Issue by Socialdarwinist** [teeuniverse/teeuniverse#190]
It would come in handy in creating consistent graphics.
![bildschirmfoto vom 2017-03-29 12-33-16](https://cloud.githubusercontent.com/assets/20446122/24450792/7fec009c-147c-11e7-9692-e2a949a88b33.png)

**Comment by necropotame** [comment]
This is made by two objects, right?

**Comment by Socialdarwinist** [comment]
No, the upper is in a tile layer. But it can also befall with two objects, for it always depends on from which side one begins the line.

**Comment by necropotame** [comment]
Ok. I just wanted to check if this orientation break was a bug or not :)
